dao/src/resolution-manager-mapping.ts

```diff
@@ -183,6 +184,8 @@ export function handleResolutionUpdated(event: ResolutionUpdated): void {
   if (resolutionEntity) {
     resolutionEntity.updateTimestamp = event.block.timestamp;
     resolutionEntity.updateBy = event.transaction.from;
+    resolutionEntity.hasQuorum = resolutionEntity.isNegative;
```

Comment:
Not sure we need here, the ultimate place where to set this would be the approved handler. After approval update won't be called anymore

Reply:
That's `handleResolutionUpdated` :)

Reply:
Yeah that's much better, I changed the code.